octo-core/lib/octocore/callbacks.rb

```diff
@@ -64,15 +69,28 @@ def update_counters(opts)
           Octo::ApiHit.increment_for(opts[:event])
         end
       end
-    end
+      # Adds user session for page_view and
+      # product_page_view to redis.
+      # It self expires in n seconds from last hit
+      def add_session(opts)
+        if opts.has_key?(:type)
+          # create a shadow key for every list
+          # this helps us in catching the event
+          # when a key expires, and then we
+          # delete the main key with the session details
+          # after playing around with the value of the key
+          Cequel::Record.redis.setex("shadow:" + opts[:enterprise].id.to_s + "_" + opts[:user].id.to_s,Octo.get_config(:session_length),"")
```

Comment:
Use single quotes when the statement inside is pure string literal and wont be evaluated.